Base/home/anon/.config/SystemServer.ini

```diff
@@ -10,11 +10,9 @@ SocketPermissions=600
 [RequestServer]
 Socket=/tmp/session/%sid/portal/request
 SocketPermissions=600
-Lazy=true
+KeepAlive=true
 Priority=low
 SystemModes=text,graphical
-MultiInstance=true
-AcceptSocketConnections=true
```

Comment:
This doesn't make any sense for RS, making RS single-instance will effectively serialise all internet downloads, and also limit the number of active global downloads to 32 (or whatever our max pipe count is these days).

Reply:
I will remove that commit, as we don't really need it since #18985.